pkg/datapath/linux/config/config.go

```diff
@@ -583,6 +583,9 @@ func (h *HeaderfileWriter) writeTemplateConfig(fw *bufio.Writer, e datapath.Endp
 			return err
 		}
 		fmt.Fprintf(fw, "#define DIRECT_ROUTING_DEV_IFINDEX %d\n", directRoutingIfIndex)
+		if option.Config.NodePortHairpin {
+			fmt.Fprintf(fw, "#define ENABLE_SKIP_FIB 1\n")
```

Comment:
Why can we skip the FIB lookup in case of a one-legged load balancer? If it's just because there's a single interface, maybe best to write it explicitly:
```go
if len(option.Config.Devices) == 1 {
```

Reply:
Yeah, it's exactly for that reason. I can write it explicitly, the NodePortHairpin is the same, but maybe it's more readable when it's spelled out as `len(option.Config.Devices) == 1`.